packages/e2e-test-utils-playwright/src/admin/create-new-post.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import type { Admin } from './';
+
+interface NewPostOptions {
+	postType?: string;
+	title?: string;
+	content?: string;
+	excerpt?: string;
+	showWelcomeGuide?: boolean;
+}
+
+/**
+ * Creates new post.
+ *
+ * @param this
+ * @param options Options to create new post.
+ */
+export async function createNewPost(
+	this: Admin,
+	options: NewPostOptions = {}
+) {
+	const query = addQueryArgs( '', {
+		post_type: options.postType,
+		post_title: options.title,
+		content: options.content,
+		excerpt: options.excerpt,
+	} ).slice( 1 );
+
+	await this.visitAdminPage( 'post-new.php', query );
+
+	await this.editor.setPreferences( 'core/edit-post', {
```

Comment:
I'm curious to know if we need to do this every time, seems wasteful to me. Could we just do this in `global-setup` with a network request? I know that the preference API doesn't seem stable in e2e tests though 😅,

Reply:
I like the idea. The only time the welcome guide needs to be enabled is when we're actually testing it.

Reply:
Is there an existing util for setting prefs via a network request? 🤔